### PR TITLE
initial dutch translations 

### PR DESCRIPTION
**Pull request by iamjochem**
possibly contains some spelling mistakes and grammatically faux-paus' - but it is a start.

Emails sent by auth0 also have i18n requirements ... is there somewhere I can go to grab files/templates so I can make a dutch-translation PR to those?

### CODE REVIEW

File: src/i18n/nl.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
+export default {
+  error: {
+    forgotPassword: {
+      "too_many_requests": "U heeft het limiet van wachtwoord aanpassings pogingen bereikt. Wacht even voor dat u het nog een keer probeert.",
```
Comment (sandrinodimattia) [inline src/i18n/nl.js]: de limiet

File: src/i18n/nl.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      "lock.fallback": "Onze excuses, er is iets fout gegaan bij de de aanvraag voor een wachtwoord aanpassing."
+    },
+    login: {
+      "blocked_user": "De gebruiker is geblokeerd.",
+      "invalid_user_password": "Verkeerde inloggegevens.",
```
Comment (sandrinodimattia) [inline src/i18n/nl.js]: I would go for "Email of wachtwoord is niet correct."

File: src/i18n/nl.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+      "lock.fallback": "Onze excuses, er is iets mis gegaan bij de poging om in te loggen.",
```
Comment (sandrinodimattia) [inline src/i18n/nl.js]: bij het inloggen

File: src/i18n/nl.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+      "lock.invalid_code": "Verkeerde code.",
+      "lock.invalid_email_password": "Verkeerde emailadres of wachtwoord.",
+      "lock.invalid_username_password": "Verkeerde gebruikersnaam of wachtwoord.",
+      "lock.network": "Server is niet bereikbaar. Check uw netwerk verbinding en probeer het nog een keer.",
+      "lock.popup_closed": "Popup window gesloten. Probeer het nog een keer.",
```
Comment (sandrinodimattia) [inline src/i18n/nl.js]: Popup venster

File: src/i18n/nl.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+  signUpLabel: "Aanmelden",
+  signUpSubmitLabel: "Aanmelden",
+  signUpTerms: "",
+  signUpWithLabel: "Aanmelden met %s",
+  socialLoginInstructions: "",
+  socialSignUpInstructions: "",
+  ssoEnabled: "Single Sign-On geactiveerd",
+  submitLabel: "Verzenden",
+  unrecoverableError: "Er ging iets mis.<br />Neem a.u.b. contact op met de technische support.",
+  usernameFormatErrorHint: "Gebruik 1-15 letters, numbers and \"_\"",
```
Comment (sandrinodimattia) [inline src/i18n/nl.js]: "nummers en"

File: src/i18n/nl.js [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    shouldContain: "Moet bevatten:",
+    specialCharacters: "Speciale karacters (e.g. !@#$%^&*)",
+    upperCase: "Hoofdletters (A-Z)"
+  },
+  passwordlessEmailAlternativeInstructions: "Anders, voer uw emailadres in om in te loggen<br/>of creëer een nieuwe gebruikersaccount",
+  passwordlessEmailCodeInstructions: "Er is een email met de code verstuurt naar %s.",
+  passwordlessEmailInstructions: "Voer uw emailadres in om in te loggen<br/>of creëer een nieuwe gebruikersaccount",
+  passwordlessSMSAlternativeInstructions: "Anders, voer uw telefoonnummer in om in te loggen<br/>of creëer een nieuwe gebruikersaccount",
+  passwordlessSMSCodeInstructions: "Er is een SMS met de code verstuurt<br/>naar %s.",
+  passwordlessSMSInstructions: "Voer uw telefoonnummer in om in te loggen<br/>of creëer een nieuwe gebruikersaccount",
```
Comment (sandrinodimattia) [inline src/i18n/nl.js]: om aan te melden of maak een nieuwe account aan

File: src/i18n/nl.js [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+  enterpriseActiveLoginInstructions: "Voer uw bedrijfs inloggegevens in op %s.",
+  failedLabel: "Gefaald!",
+  forgotPasswordAction: "Wachtwoord vergeten??",
+  forgotPasswordInstructions: "Geef uw emailadres op. Wij sturen een email voor het resetten van uw wachtwoord.",
+  forgotPasswordSubmitLabel: "Verstuur email",
+  invalidErrorHint: "Ongeldig",
+  lastLoginInstructions: "U heeft laatst ingelogd met",
+  loginAtLabel: "Inloggen op %s",
+  loginLabel: "Inloggen",
+  loginSubmitLabel: "Inloggen",
```
Comment (sandrinodimattia) [inline src/i18n/nl.js]: Aanmelden

File: src/i18n/nl.js [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+  loginWithLabel: "Inloggen met %s",
```
Comment (sandrinodimattia) [inline src/i18n/nl.js]: Aanmelden

File: src/i18n/nl.js [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+  enterpriseLoginIntructions: "Login met uw bedrijfs inloggegevens.",
+  enterpriseActiveLoginInstructions: "Voer uw bedrijfs inloggegevens in op %s.",
+  failedLabel: "Gefaald!",
+  forgotPasswordAction: "Wachtwoord vergeten??",
+  forgotPasswordInstructions: "Geef uw emailadres op. Wij sturen een email voor het resetten van uw wachtwoord.",
+  forgotPasswordSubmitLabel: "Verstuur email",
+  invalidErrorHint: "Ongeldig",
+  lastLoginInstructions: "U heeft laatst ingelogd met",
+  loginAtLabel: "Inloggen op %s",
+  loginLabel: "Inloggen",
```
Comment (sandrinodimattia) [inline src/i18n/nl.js]: Aanmelden

File: src/i18n/nl.js [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+  emailInputPlaceholder: "iets@example.com",
+  enterpriseLoginIntructions: "Login met uw bedrijfs inloggegevens.",
+  enterpriseActiveLoginInstructions: "Voer uw bedrijfs inloggegevens in op %s.",
+  failedLabel: "Gefaald!",
+  forgotPasswordAction: "Wachtwoord vergeten??",
+  forgotPasswordInstructions: "Geef uw emailadres op. Wij sturen een email voor het resetten van uw wachtwoord.",
+  forgotPasswordSubmitLabel: "Verstuur email",
+  invalidErrorHint: "Ongeldig",
+  lastLoginInstructions: "U heeft laatst ingelogd met",
+  loginAtLabel: "Inloggen op %s",
```
Comment (sandrinodimattia) [inline src/i18n/nl.js]: Aanmelden

File: src/i18n/nl.js [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+  databaseAlternativeSignUpInstructions: "of",
+  emailInputPlaceholder: "iets@example.com",
+  enterpriseLoginIntructions: "Login met uw bedrijfs inloggegevens.",
+  enterpriseActiveLoginInstructions: "Voer uw bedrijfs inloggegevens in op %s.",
+  failedLabel: "Gefaald!",
+  forgotPasswordAction: "Wachtwoord vergeten??",
+  forgotPasswordInstructions: "Geef uw emailadres op. Wij sturen een email voor het resetten van uw wachtwoord.",
+  forgotPasswordSubmitLabel: "Verstuur email",
+  invalidErrorHint: "Ongeldig",
+  lastLoginInstructions: "U heeft laatst ingelogd met",
```
Comment (sandrinodimattia) [inline src/i18n/nl.js]: U bent het laatst aangemeld met?

**Comment by iamjochem** [comment]
hi @sandrinodimattia, I've implemented your suggested changes/fixes ... any word on being able to translate email templates?

**Comment by hzalaz** [comment]
@iamjochem thanks. For email templates you can check https://auth0.com/docs/email/templates#multilingual-email-templates

**Comment by iamjochem** [comment]
@hzalaz - np, glad to see it merged, thanks for pointing me at the template-related functionality.

File: src/i18n/nl.js [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+    forgotPassword: {
+      "too_many_requests": "U heeft het limiet van wachtwoord aanpassings pogingen bereikt. Wacht even voor dat u het nog een keer probeert.",
+      "lock.fallback": "Onze excuses, er is iets fout gegaan bij de de aanvraag voor een wachtwoord aanpassing."
+    },
+    login: {
+      "blocked_user": "De gebruiker is geblokeerd.",
+      "invalid_user_password": "Verkeerde inloggegevens.",
+      "lock.fallback": "Onze excuses, er is iets mis gegaan bij de poging om in te loggen.",
+      "lock.invalid_code": "Verkeerde code.",
+      "lock.invalid_email_password": "Verkeerde emailadres of wachtwoord.",
```
Comment (ToonDC) [inline src/i18n/nl.js]: e-mailadres

File: src/i18n/nl.js [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+      "lock.invalid_username_password": "Verkeerde gebruikersnaam of wachtwoord.",
+      "lock.network": "Server is niet bereikbaar. Check uw netwerk verbinding en probeer het nog een keer.",
+      "lock.popup_closed": "Popup window gesloten. Probeer het nog een keer.",
+      "lock.unauthorized": "Authorisatie geweigerd. Probeer het nog een keer.",
+      "password_change_required": "U moet uw wachtwoord aanpassen omdat dit de eerste keer is dat u inlogt of omdat uw wachtwoord is verlopen.", // TODO: verify error code
+      "password_leaked": "Deze login is geblokkeerd omdat uw wachtwoord is gelekt op een andere website. We hebben een email verstuurd met instructies voor het deblokkeren.",
```
Comment (ToonDC) [inline src/i18n/nl.js]: e-mail

File: src/i18n/nl.js [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+      "too_many_attempts": "Uw account is geblokkeerd als gevolg van herhaalde pogingen om in te loggen."
+    },
+    passwordless: {
+      "bad.email": "Het emailadres is ongeldig",
```
Comment (ToonDC) [inline src/i18n/nl.js]: e-mailadres

File: src/i18n/nl.js [hunk 15]
```diff
@@ -0,0 +1,102 @@
...
+      "password_dictionary_error": "Wachtwoord is een te bekend woord.",
+      "password_no_user_info_error": "Wachtwoord is gebaseerd op gebruikers informatie.",
+      "password_strength_error": "Wachtwoord is niet sterk genoeg.",
+      "user_exists": "Gebruiker bestaat al.",
+      "username_exists": "Gebruikersnaam bestaat al."
+    }
+  },
+  success: { // success messages show above the form or in a confirmation pane
+    logIn: "Dank u voor het inloggen.",
+    forgotPassword: "Wij hebben een email verstuurd voor het resetten van uw wachtwoord.",
```
Comment (ToonDC) [inline src/i18n/nl.js]: e-mail

File: src/i18n/nl.js [hunk 16]
```diff
@@ -0,0 +1,102 @@
...
+  databaseEnterpriseLoginInstructions: "",
+  databaseEnterpriseAlternativeLoginInstructions: "of",
+  databaseSignUpInstructions: "",
+  databaseAlternativeSignUpInstructions: "of",
+  emailInputPlaceholder: "iets@example.com",
+  enterpriseLoginIntructions: "Login met uw bedrijfs inloggegevens.",
+  enterpriseActiveLoginInstructions: "Voer uw bedrijfs inloggegevens in op %s.",
+  failedLabel: "Gefaald!",
+  forgotPasswordAction: "Wachtwoord vergeten??",
+  forgotPasswordInstructions: "Geef uw emailadres op. Wij sturen een email voor het resetten van uw wachtwoord.",
```
Comment (ToonDC) [inline src/i18n/nl.js]: e-mailadres
e-mail

File: src/i18n/nl.js [hunk 17]
```diff
@@ -0,0 +1,102 @@
...
+  forgotPasswordSubmitLabel: "Verstuur email",
```
Comment (ToonDC) [inline src/i18n/nl.js]: e-mail

File: src/i18n/nl.js [hunk 18]
```diff
@@ -0,0 +1,102 @@
...
+    identicalChars: "Niet meer dan %d identieke karacters op een rij (e.g., \"%s\" is niet toegestaan)",
+    nonEmpty: "Wachtwoord mag niet leeg zijn",
+    numbers: "Nummers (i.e. 0-9)",
+    lengthAtLeast: "Minstens %d karacters lang",
+    lowerCase: "kleine letters (a-z)",
+    shouldContain: "Moet bevatten:",
+    specialCharacters: "Speciale karacters (e.g. !@#$%^&*)",
+    upperCase: "Hoofdletters (A-Z)"
+  },
+  passwordlessEmailAlternativeInstructions: "Anders, voer uw emailadres in om in te loggen<br/>of creëer een nieuwe gebruikersaccount",
```
Comment (ToonDC) [inline src/i18n/nl.js]: e-mailadres

File: src/i18n/nl.js [hunk 19]
```diff
@@ -0,0 +1,102 @@
...
+  passwordlessEmailCodeInstructions: "Er is een email met de code verstuurt naar %s.",
```
Comment (ToonDC) [inline src/i18n/nl.js]: e-mail

File: src/i18n/nl.js [hunk 20]
```diff
@@ -0,0 +1,102 @@
...
+  passwordlessEmailInstructions: "Voer uw emailadres in om in te loggen<br/>of creëer een nieuwe gebruikersaccount",
```
Comment (ToonDC) [inline src/i18n/nl.js]: e-mailadres

File: src/i18n/nl.js [hunk 21]
```diff
@@ -0,0 +1,102 @@
...
+  signUpTerms: "",
+  signUpWithLabel: "Aanmelden met %s",
+  socialLoginInstructions: "",
+  socialSignUpInstructions: "",
+  ssoEnabled: "Single Sign-On geactiveerd",
+  submitLabel: "Verzenden",
+  unrecoverableError: "Er ging iets mis.<br />Neem a.u.b. contact op met de technische support.",
+  usernameFormatErrorHint: "Gebruik 1-15 letters, numbers and \"_\"",
+  usernameInputPlaceholder: "uw gebruikersnaam",
+  usernameOrEmailInputPlaceholder: "gebruikersnaam/emailadres",
```
Comment (ToonDC) [inline src/i18n/nl.js]: e-mailadres

**Comment by hzalaz** [comment]
Thanks @ToonDC the pull request is already merged, can you submit a new one with those fixes?

**Comment by ToonDC** [comment]
@hzalaz pull request can be found here https://github.com/auth0/lock/pull/629.

**Pull request merged**